src/core/ledger/policies/immediate.js

```diff
@@ -65,11 +66,13 @@ export const immediateConfigParser: P.Parser<ImmediatePolicy> = P.object({
   numIntervalsLookback: P.number,
 });
 
-export const immediatePolicyParser: P.Parser<ImmediatePolicy> = P.object({
-  policyType: P.exactly(["IMMEDIATE"]),
-  budget: grainParser,
-  numIntervalsLookback: P.number,
-});
+export const immediatePolicyParser: P.Parser<ImmediatePolicy> = P.object(
+  {
+    policyType: P.exactly(["IMMEDIATE"]),
+    budget: grainParser,
+  },
+  {numIntervalsLookback: P.number}
+);
```

Comment:
Why did this change? What are the semantics going on here?

Reply:
Maybe have the `numIntervalsLookback` parser default to 1 if missing.

Reply:
Moving numIntervalsLookback to the second param makes it optional instead of required, which prevents breaking old ledger entries that didn't have it defined